src/main/java/uk/gov/hmcts/reform/sscs/handler/SscsCaseDataHandler.java

```diff
@@ -43,7 +52,16 @@ public CallbackResponse handle(CaseResponse caseValidationResponse,
 
                 log.info("Case created with caseId {} from exception record id {}", caseId, exceptionRecordId);
 
-                roboticsHandler.handle(caseValidationResponse, caseId, eventId);
+                if (eventId.equals(caseEvent.getCaseCreatedEventId())) {
+                    roboticsHandler.handle(caseValidationResponse, caseId);
+
+                    if (sendToDwpFeature) {
+                        log.info("About to update case with sendToDwp event for id {}", caseId);
+                        caseDataHelper.updateCase(caseValidationResponse.getTransformedCase(), token.getUserAuthToken(), token.getServiceAuthToken(), token.getUserId(), SEND_TO_DWP.getCcdType(), caseId);
```

Comment:
Is it intentional that there we're not catching an exception here and at ` roboticsHandler.handle(caseValidationResponse, caseId);` ?

When there's an error should it rollback and re-try?